doc(administrate/activity): update maintenant redeployment description (#350)
CleverCloud/documentation: content/doc/administrate/activity.md

![Re-deployment triggered by the monitoring](/images/doc/monitoring.png)

- `Unreachable`: The monitoring servers couldn't open a TCP connection with at least one monitoring agent hosted on instances of the application.

> - `Timeout`: The monitoring servers has succeed to open TCP connections on all monitoring agents hosted on instances of the application, but at least one did not send its probes in time.

Reviewer: ```suggestion
- `Timeout`: The monitoring servers has succeed to open TCP connections on all monitoring agents hosted on instances of the application, but at least one didn't send its probes in time.
```

> - `Scaling`: We trigger a new deployment with more instances (virtual machines) or more resources of the application as we detect that the consumption is greater that the determined threshold on memory or cpu usage. For further details, see [Scalability](../scalability)

Reviewer: ```suggestion
- `Scaling`: Clever Cloud triggers a new deployment with more instances (virtual machines) or more resources of the application as the platform detects that the consumption is greater that the determined threshold on memory or CPU usage. For further details, see [Scalability](../scalability)
```

> - `Scaling`: We trigger a new deployment with more instances (virtual machines) or more resources of the application as we detect that the consumption is greater that the determined threshold on memory or cpu usage. For further details, see [Scalability](../scalability)

Reviewer: ```suggestion
- `Scaling`: Clever Cloud triggers a new deployment with more instances (virtual machines) or more resources of the application as the platform detects that the consumption is greater that the determined threshold on memory or CPU usage. For further details, see [Scalability](../scalability)
```
